config/locales/en.yml

```diff
@@ -1995,7 +1995,7 @@ en:
       visible_to_registered_users: "(visible to users that have logged in)"
       default_in_listing: "(used only as a default when creating a new listing)"
       invisible: "(not shown to other users)"
-      image_is_processing: "We are processing your profile picture, it will be visible in a few seconds."
+      image_is_processing: "Hang on tight. We are processing your profile picture. It will be ready in a minute or two."
```

Comment:
Profile picture processing was quick, but processing a front page cover image took a minute or two for me. Thought it would be better to underpromise and overdeliver here, just in case it does sometimes take longer.